code/game/gamemodes/changeling/powers/tiny_prick.dm

```diff
@@ -118,6 +117,8 @@
 		spawn(10)
 			user.real_name = NewDNA.real_name
 			NewDNA.transfer_identity(C, transfer_SE=1)
+			if(!istype(selected_dna.dna.species, /datum/species/human))
+				C.humanize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSE) //To keep lingstings from adding the monkey disability.
```

Comment:
Doesn't happen because the humanize() call comes after the transfer_identity() call.